ui/exceptions.py

```diff
@@ -0,0 +1,6 @@
+""" Custom exceptions"""
+
+
+class MoiraException(Exception):
+    """Custom exception to be used when something goes wrong with Moira API calls"""
+    pass
```

Comment:
Can you remove the `pass` here? It's not necessary since the docstring does the same thing here

Reply:
OK